style.css

```diff
@@ -322,106 +322,13 @@
     letter-spacing: 0.03rem;
   }
 
-  .project_container_1 {
+  .project_container {
```

Comment:
- [ ] On the desktop version, the text in the first popup has an overflow as shown in the image below. I suggest that you make the images the same size across all the popups and also give the same text area to all popups for uniformity and a more professional design

### Your design
<img width="1223" alt="Screenshot 2023-07-02 at 00 21 36" src="https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/64774936/c3efd43e-171b-48e4-a70f-9d090e9d7ff9">

### Figma design

![Screenshot 2023-07-02 at 00 21 36 (2)](https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/64774936/45821188-f1aa-4087-842a-86884d92c2f9)